.github/workflows/publish-to-pypi.yml

```diff
@@ -13,27 +13,34 @@ on:
 
 jobs:
   deploy:
-
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v3
-      with:
-        submodules: recursive
-    - name: Set up Python
-      uses: actions/setup-python@v2
-      with:
-        python-version: '3.7'
-        #cache: 'pip'
-    - name: Install build dependencies
-      run: |
-        python -m pip install --upgrade pip
-        pip install setuptools wheel twine build
-    - name: Build package
-      run: |
-        python -m build
-     
-    - name: Publish to PyPI
-      uses: pypa/gh-action-pypi-publish@release/v1
-      with:
-        password: ${{ secrets.PYPI_API_TOKEN }}
+      - uses: actions/checkout@v3
+        with:
+          submodules: recursive
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.7"
```

Comment:
`3.7`? Not `3.8`?

Reply:
> `3.7`? Not `3.8`?

No clue. Anything about pushing to pypi that would matter there or should we just bump it?

Reply:
@dmarx any thoughts here?

Reply:
I think when we set this up, google colab was pinned to 3.7 and I wanted to make sure the pypi package was compatible. I'm pretty sure colab uses 3.8 now, so we can probably bump the version up. will double check tomorrow.

Reply:
Yup - 
![image](https://user-images.githubusercontent.com/232217/207666652-37fe3f76-940a-49e1-8743-39f3a3ad6621.png)